src/components/Search/SearchFrameworkComp.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+
+import InfoIcon from '@mui/icons-material/InfoOutlined';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import { SelectChangeEvent } from '@mui/material/Select';
+import Tooltip from '@mui/material/Tooltip';
+import { style, cssRule } from 'typestyle';
+
+import { useAppSelector } from '../../hooks/app';
+import { Strings } from '../../resources/Strings';
+import {
+  Spacing,
+  ButtonsLightRaw,
+  ButtonsDarkRaw,
+  DropDownMenuRaw,
+  DropDownItemRaw,
+} from '../../styles';
+import type { Framework } from '../../types/types';
+import FrameworkDropdown from '../Shared/FrameworkDropdown';
+
+const strings = Strings.components.searchDefault.sharedCollasped.framework;
+
+interface FrameworkDropdownProps {
+  frameworkId: Framework['id'];
+}
+
+function SearchFramework({ frameworkId }: FrameworkDropdownProps) {
```

Comment:
nit: for the matter of consistency and finding easily the file when grepping, can you please call the component `SearchFrameworkDropdown`, as well as where it's used, and also rename the file itself?